### PR TITLE
Gives moderators respawn and change name from VV.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Approved by the headmins.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Review by Geevies — APPROVED** [review]
hey uhh can i uhhh

uhh

uhh

i joined as the wrong


can i get a respawn

File: html/changelogs/mattatlas-mod.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - admin: "Moderators may now respawn players and change verbs from VV."
```
Comment (NonQueueingMatt) [inline html/changelogs/mattatlas-mod.yml:41]:
```suggestion
  - admin: "Moderators may now respawn players and change names from VV."
```

**Comment by NonQueueingMatt** [comment]
!review

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**